csv2json/csv2json.py

```diff
@@ -0,0 +1,58 @@
+import  json
+import csv
+
+
+def readcsv(fname:str) -> (list):
+    memdata = []
+    with open(fname, newline='', encoding='utf-8') as csvfile:    
+        for item in csvfile:
+            item = item.encode('utf-8').decode('utf-8-sig').strip()
+            data = item.split(',')
+            memdata.append(data)
+            #print(memdata)
+    return memdata
+
+def csv2json(uniquefname:str, commonfname:str) -> (dict):

```

Comment:
Same as line 5. Let's do following update. 
```suggestion
from typing import Dict, List
#...
def csv2json(uniquefname:str, commonfname:str) -> Dict[str, List[str]]:
```

Reply:
I prefer using `csv_to_json`.